Wave/Wave/RowType.swift

```diff
@@ -13,4 +13,5 @@ public protocol RowType {
     
     func registerCells(tableView: UITableView)
     func configureCell(tableViewCell: UITableViewCell)
+    func didSelectCell(tableViewCell: UITableViewCell, tableView: UITableView, indexPath: NSIndexPath)
```

Comment:
I'd say we should cut this method down to only take the params it needs. Guessing that's either just the cell or the row (as an int)?